### ISSUE TITLE
OCAT- OCvalidate

### ISSUE THREAD

**Issue by esafeddie** [ic005k/OCAuxiliaryTools#408]
OCvalidate app needs updating to add the missing line in the newly released Opencore config.plist
(See attached example)
![Screenshot 2024-10-07 at 13 43 33](https://github.com/user-attachments/assets/d4fbebb5-2caa-4382-bc9f-beb288d1f22c)

**Comment by coutureone** [comment]
I had the same problem. Did you solve it

**Comment by ic005k** [comment]
@coutureone  Here is a temporary solution, please watch and download the latest beta version:

https://github.com/ic005k/OCAuxiliaryTools/actions

**Comment by esafeddie** [comment]
Problem solved thanks to the Dev.

**Comment by coutureone** [comment]
> Problem solved thanks to the Dev.

thank  you，i had solved it

**Comment by fabiosun** [comment]
@ic005k hi
error is not shown anymore but i cant see on OCAT Uefi interface Unload tab
Or maybe am i missing something?
Thank you

**Comment by ic005k** [comment]
> error is not shown anymore but i cant see on OCAT Uefi interface Unload tab
> Or maybe am i missing something?

Yes, but I don't know how to use this Unload at the moment, so I don't display things like tables on the OCAT interface.

If you know how to use it, please give an example, I will refer to your example. (Since I haven't used a Hackintosh system in a long time, I can't test these things)

**Comment by fabiosun** [comment]
> > error is not shown anymore but i cant see on OCAT Uefi interface Unload tab
> > Or maybe am i missing something?
> 
> Yes, but I don't know how to use this Unload at the moment, so I don't display things like tables on the OCAT interface.
> 
> If you know how to use it, please give an example, I will refer to your example. (Since I haven't used a Hackintosh system in a long time, I can't test these things)

`Unload
Type: plist array
Failsafe: Empty
Description: Unload specified firmware drivers.~
Io be filled with plist string entries containing the names of firmware drivers to unload before loading the Drivers section. This setting is typically only reguired if a user-provided driver is a variant of an existing system firmware driver, and if the new driver would detect itself as partially loaded, or otherwise fail to operate correctly. if the old driver is not unloaded first.
Warning: Unloading system firmware drivers is usually not required and not recommended. Poorly written drivers may crash when unloaded, or cause subseguent crashes leg by allowing themselves to be unloaded even though they have active dependencies). However standard VEFI network stack drivers should unload cleanly.
Note L: See SysReport Drivers Driver ImageNames. txt for the list of drivers which this option can attempt to unload. The relevant name is the driver component name. Drivers are only listed if they implement DriverBindingProtecol and Loaded ImageProtocol, and have an available component name.
Note 2: The NVRAM Lang and Platformlang variables are ignored when determining the driver component names recognised by this option, and listed in the Sysheport file. This is in order to make unloading images stable across changes in these variables. The UEFI Shell dh command takes account of these variables, so in some circumstances may display different driver component names from those listed for this option, unless these variables are cleared`
maybe it is not only hackintosh related but i could be wrong

**Comment by ic005k** [comment]
> maybe it is not only hackintosh related but i could be wrong

Please download the latest OCAT development version to test, I have done further work on Unload:

https://github.com/ic005k/OCAuxiliaryTools/actions

**Comment by fabiosun** [comment]
> > maybe it is not only hackintosh related but i could be wrong
> 
> Please download the latest OCAT development version to test, I have done further work on Unload:
> 
> https://github.com/ic005k/OCAuxiliaryTools/actions

<img width="1062" alt="Screenshot 2024-10-10 at 1 41 37 pm" src="https://github.com/user-attachments/assets/065f1753-3c91-426f-9b00-2a9c169147b4">
Do you mean this?

**Comment by fabiosun** [comment]
> > Do you mean this?
> 
> Yes, but I don't know what to fill in, that is, how to use Unload.

it seems you did well @ic005k 
https://www.hackintosh-forum.de/forum/thread/42354-opencore-sammelthread-hilfe-und-diskussion/?postID=796279#post796279

![image](https://github.com/user-attachments/assets/e98c7cde-c8d2-41d8-9984-a40893e043e5)
![image](https://github.com/user-attachments/assets/b1831d56-88a8-4ac4-8b5f-d44f088872c0)

thanks at cobanramo from hackitosh-forum de

**Comment by KillingAlisa** [comment]
> > @coutureone Here is a temporary solution, please watch and download the latest beta version:
> > In general, the latest beta version will be shown in the link below, please feel free to leave a message if you have any questions.
> > https://github.com/ic005k/OCAuxiliaryTools/actions
> 
> thank you, i had solved it


all Artifacts are expired
